pkg/ddc/alluxio/port_parser_test.go

```diff
@@ -1,3 +1,16 @@
+/*
+Copyright 2021 The Fluid Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
```

Comment:
I suggest you should follow the format of https://github.com/fluid-cloudnative/fluid/blob/master/pkg/ddc/juicefs/const.go#L1-L17